docs/_source/conf.py

```diff
@@ -42,6 +42,7 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
+    'sphinx_autodoc_typehints',
```

Comment:
This allows sphinx to better "understand" type annotations and populate those types in the docs. Without this, sphinx was logging warnings when I'd generate docs and wasn't generating docs that were properly indicating the types.